Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,26 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function getNameByHouse(hogwarts, house) {
+  // filter hogwarts to include people from specific house only
+  const houseMembers = hogwarts.filter(person => person.house === house);
```

Comment:
I like your filter function here. You can actually chain your code this way:

hogwarts.filter().forEach();

While your current solution is more verbose and valid, sometimes you may choose to use the chaining approach. Most of the time it is a matter of code style or personal choice.